src/modules/views/MissionControl.jsx

```diff
@@ -50,10 +50,13 @@ const useStyles = makeStyles(theme => ({
 }))
 
 const MissionControl = props => {
-  const [state, dispatch] = useContext(AppContext);
-  
+  const [state, dispatch] = useContext(AppContext);  
   const classes = useStyles()
 
+  useEffect(() => {
+    console.log('effect');
+  })
+
   const makeMissionList = () => {
```

Comment:
Do we want a user to be able to view/complete missions if they aren't logged in/signed up?